RecoTauTag/RecoTau/src/AntiElectronIDMVA6.cc

```diff
@@ -57,7 +57,6 @@ AntiElectronIDMVA6::AntiElectronIDMVA6(const edm::ParameterSet& cfg)
   Var_woGwGSF_Endcap_ = new Float_t[23];
   Var_wGwGSF_Endcap_ = new Float_t[31];
```

Comment:
compared to the master/106X versions, this file is missing removal of 
```c++
#include "MagneticField/Engine/interface/MagneticField.h"
#include "MagneticField/Records/interface/IdealMagneticFieldRecord.h"
```

Reply:
Also indentation is different => synchronized.